pkg/monitoring/migration/vmi-migration-phase-transitions.go

```diff
@@ -115,6 +115,7 @@ func newVMIMigrationPhaseTransitionTimeFromCreationHistogramVec(informer cache.S
 	histogramVec := prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Name:    "kubevirt_vmi_migration_phase_transition_time_from_creation_seconds",
+			Help:    "Histogram of VM migration phase transitions duration from creation time in seconds",
```

Comment:
nit: we usually do not include the type again in the help text, it already shows on the documentation and on the tools. Maybe `VM migration phase transitions duration from creation time in seconds` is enough.

Reply:
I think it should be visible in the Help text, since the actual metrics that are created from this are `counters` and in the UI, like Grafana, we need to add this information or we might not understand its an histogram.

Reply:
@machadovilaca Thoughts?